R/lazy.R

```diff
@@ -1,4 +1,4 @@
-as_lazy_duckplyr_df <- function(x, allow_materialization, n_rows, n_cells) {
+as_funneled_duckplyr_df <- function(x, allow_materialization, n_rows, n_cells) {
```

Comment:
The file names should be changed too.

Reply:
https://github.com/tidyverse/duckplyr/pull/509